MediaAdminBundle/Resources/views/Tree/menu_macros_modal.html.twig

```diff
@@ -16,8 +16,10 @@
                href="#"
                data-url="{{ path('open_orchestra_media_admin_folder_new', {'parentId': parentId}) }}"
            >
-           <i class="fa fa-file"></i>
-                    {{ 'open_orchestra_media_admin.left_menu.tree.new_media_folder'|trans }}
+                   <span class="btn btn-success">
+                        <i class="fa fa-file"></i>
+                            {{ 'open_orchestra_media_admin.left_menu.tree.new_media_folder'|trans }}
+                   </span>
```

Comment:
You have an ident problem the closing a, div and li does not have the same padding as the opening tags